src/routes/index.ts

```diff
@@ -4,5 +4,6 @@ export const router = express.Router();
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('index', { title: 'Express' });
+  res.render('character', { testejs: 'hi!' });
```

Comment:
We can't render 2 pages in one function. Move `res.render('character', { testejs: 'hi!' });` to another router.get function with path 'character'